ddtrace/contrib/pytest/_plugin_v2.py

```diff
@@ -118,30 +118,32 @@ def _handle_collected_coverage(test_id, coverage_collector) -> None:
     CISuite.add_coverage_data(test_id.parent_id, coverage_data)
 
 
-class _PytestDDTracePluginV2:
-    """Plugin hooks (and some convenience methods) for the pytest plugin
-
-    This plugin leverages the external CI Visibility API which behaves in a stateless way, where IDs are passed to the
-    API calls, so each method is independent of all others, and may re-compute item ids.
+def _disable_ci_visibility():
+    try:
+        disable_ci_visibility()
+    except:  # noqa: E722
+        log.debug("encountered error during disable_ci_visibility", exc_info=True)
 
-    Methods are ordered in this class mostly in the order they are called during a test run.
-    """
 
-    @staticmethod
-    def pytest_configure(config: pytest.Config) -> None:
+def pytest_configure(config: pytest.Config) -> None:
+    try:
         if is_enabled(config):
             unpatch_unittest()
             take_over_logger_stream_handler()
             enable_ci_visibility(config=dd_config.pytest)
         if _is_pytest_cov_enabled(config):
             patch_coverage()
+    except:  # noqa: E722            log.warning("encountered error during configure, disabling Datadog CI Visibility", exc_info=True)
```

Comment:
@romainkomorndatadog Missing newline? :eyes:

Reply:
You are as eagle-eyed as I am potato-eyed. :(